Datasets/data-connect-dataset-teamschannelactivity.md

```diff
@@ -0,0 +1,90 @@
+- It can also be used with TeamsChannelDetails_v0 to get more information regarding the channels queried here.
+
+## Definitions
+
+- **Signal:**  A Substrate signal is a high-value event that captures a user's behavior. A signal can be either active (like joining a meeting) or passive (like a location change).
+
+
+## Schema
+
+| Name  | Type  |  Description  |  FilterOptions  |  FilterType  |
+| ----------- | ----------- | ----------- | ----------- | ----------- |
+| SignalType | string | Name of the signal. | No | None |
+| Id | string | The unique identifier of the Signal. | No | None |
+| StartTime | datetime | The date and time when action happens. | No | None |
+| CreationTime | datetime | The date and time when the signal was created. | Yes | Date |
+| EndTime | datetime | The date and time when action ends. If a signal is a point-in-time action, it is same as StartTime. If the action lasts a time period, this is the action end time captured on client. | No | None|
+| ItemType | string | The type of item being acted on. | No | None |
+| AppName | string | The end user client name being used to generate the signal, E.g. "Unknown", "OutlookMobile", "OWA", "OutlookDesktop", etc. | No | None |
+| AadTenantId | string | AadTenantId of the actor (signal generator). | No | None |
+| ThreadName | string | The Channel’s Name. Among the properties defined by the signal producer. | No | None |                
+| PriorThreadName | string | Older Channel’s Name (in case the signal was for renaming the channel). Among the properties defined by the signal producer. | No |  None |               
+| TeamGroupId | string | Unique GroupId of the teams channel. Among the properties defined by the signal producer. | No | None |                
+| TeamName | string | The team name owning the Channel. Among the properties defined by the signal producer. | No | None |             
+| MembersAdded | string | A list of members added (in case the signal was AddToChannel or Join Channel). Among the properties defined by the signal producer. | No | None |               
+| MembersRemoved | string | A list of members removed (in case the signal was RemoveFromChannel or LeaveChannel). Among the properties defined by the signal producer. | No | None |        
+| ThreadMembers | string | A list of members in the channel (in case the signal was RenameChannel). Among the properties defined by the signal producer. | No | None |
+
+## JSON representation
+
+```json
+{
+"SignalType": "string", 
+"Id": "string", 
+"StartTime": "datetime", 
+"CreationTime": "datetime", 
+"EndTime": "datetime", 
+"ItemType": "string", 
+"AppName": "string", 
+"AadTenantId": "string", 
+"ThreadName": "string", 
+"PriorThreadName": "string", 
+"TeamGroupId": "string", 
+"TeamName": "string", 
+"MembersAdded": "string", 
+"MembersRemoved": "string", 
+"ThreadMembers": "string", 
+"puser": "String (identifier)", 
+"ptenant": "String (identifier)"
+}
+```
+
+## Sample
+
+```json
```

Comment:
Is it possible to add some rows with sample values of PriorThreadName, MembersRemoved, ThreadMembers?